addon/helpers/capitalize.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+const {
+  Helper: { helper },
+  String
+} = Ember;
+
+export function capitalize([string]) {
+  return String.capitalize(string);
```

Comment:
We should add a guard for `string` – if it isn't actually a `String`, the Ember method will explode. Please also add a test case for when the argument is undefined